docs/modules/migrate/pages/evaluation-tool.adoc

```diff
@@ -0,0 +1,246 @@
+   + "    </network>\n"
+   + "</hazelcast-client>\n";
+----
+. Use the tool's `remoteMap()` method to specify the map in the 3.x cluster (source)
+by providing its name and the client configuration mentioned above:
++
+[source,java]
+----
+BatchSource<Entry<Object, Object>> source = Hz3Sources.remoteMap(sourceMap, HZ3_CLIENT_CONFIG);
+----
+. Specify the map in the 5.x cluster (target) to which the 3.x map data will be copied:
++
+[source,java]
+----
+Sink<Entry<Object, Object>> sink = Sinks.map(targetMap);
+----
+. Create the pipeline to copy the 3.x map data to 5.x map:
++
+[source,java]
+----
+Pipeline p = Pipeline.create();
+p.readFrom(source)
+ .writeTo(sink);
+----
+. Configure the custom classpaths to separate the libraries for the 3.x member/client and evaluation tool (connector):
++
+[source,java]
+----
+List<String> jars = new ArrayList<>();
+jars.add("hazelcast-3.12.12.jar");
+jars.add("hazelcast-client-3.12.12.jar");
+jars.add("hazelcast-3-connector-impl.jar");
+JobConfig config = new JobConfig();
+config.addCustomClasspaths(source.name(), jars)
+----
++
+Above libraries (JAR files) should exist in the directory specified by the
+`hazelcast.jet.custom.lib.dir` property. This is already set up for the distributions
+of 3.x member/client and connector.
+. Copy the map data from 3.x to 5.x using a Jet engine job:
++
+[source,java]
+----
+Job job = hz.getJet().newJob(p, config);
+job.join();
+----
+
+== Copy Data From 5.x to 3.x
+
+Using the evaluation tool, you can also copy data from a 5.x map to a 3.x map. Follow the below steps.
+
+. You need to have a client to connect to the remote (3.x) cluster from 5.x. For this, a client
+configuration should be provided to identify the 3.x cluster. This configuration must conform to the 3.x schema;
+it should be provided as an XML document in a string. See the example below.
++
+[source,java]
+----
+String HZ3_CLIENT_CONFIG =
+"<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
+   + "<hazelcast-client xmlns=\"http://www.hazelcast.com/schema/client-config\"\n"
+   + "                  xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\n"
+   + "                  xsi:schemaLocation=\"http://www.hazelcast.com/schema/client-config\n"
+   + "                  http://www.hazelcast.com/schema/client-config/hazelcast-client-config-3.12.xsd\">\n"
+   + "\n"
+   + "    <network>\n"
+   + "        <cluster-members>\n"
+   + "            <address>127.0.0.1:3210</address>\n"
+   + "        </cluster-members>\n"
+   + "    </network>\n"
+   + "</hazelcast-client>\n";
+----
+. Specify the map in the 5.x cluster (source):
++
+[source,java]
+----
+BatchSource<Entry<Object, Object>> source = Sources.map(sourceMap);
+----
+. Specify the map in the 3.x cluster (target) to which the 5.x map data will be copied,
+by providing its name and the client configuration mentioned above:
++
+[source,java]
+----
+Sink<Entry<Object, Object>> sink = Hz3Sinks.map(targetMap, HZ3_CLIENT_CONFIG);
```

Comment:
I see we have a mix of different names here, this should be `remoteMap`. Could you change this here, I sill send a PR to fix this in the API.